### ISSUE TITLE
Cloning characters 

### ISSUE THREAD

**Issue by HyperCriSiS** [gondwanagenesis/Ada#2]
Hi,

Do you also plan some kind of character replication?
I would like to create a voice assistant which is able to emulate different characters from TV-Shows for example 😀
Your project description sounds incredible. Can't await to try this out. 

Best regards
Hyper

**Comment by gondwanagenesis** [comment]
Yes, that sounds like a great idea, it would be pretty easy to clone them. if you want to help,  the thing to do here would be to Add in the functionality to read out loud the responding text and also be able to input text by speech, period. Whisper API would probably be the best for this, but maybe you can find another option. If you add this in as functionality, that would be the first step. Then later on we could modify it to have different character voices.

**Comment by HyperCriSiS** [comment]
Hey 🖖 happy new year!
Unfortunately I am no dev :(
If you keep working in this project, I think it would be best if you concentrate on the core features and just add an easy possibility to exchange input/output with other AI solutions.
For example localai.io or other projects could just be used for the voice and other parts and ADA would be the brain.

**Comment by HyperCriSiS** [comment]
Regarding your roadmap, I want to suggest that you use other projects which already can do that and just attach ADA to it, or what was your plan?
